test/Microsoft.AspNet.Mvc.Core.Test/Rendering/HtmlHelperFormExtensionsTest.cs

```diff
@@ -276,7 +276,7 @@ public static TheoryData<object> RouteValuesDataSet
         public void BeginFormWithNoParameters_CallsHtmlGeneratorWithExpectedValues()
         {
             // Arrange
-            var tagBuilder = new TagBuilder(tagName: "form", htmlEncoder: new HtmlEncoder());
+            var tagBuilder = new TagBuilder(tagName: "form", htmlEncoder: new NullTestEncoder());
```

Comment:
It is tested here https://github.com/aspnet/Mvc/pull/2420/files#diff-8cd78f293ecf9bb7f3a377fc18b05d39R162